CMakeLists.txt

```diff
@@ -71,12 +71,7 @@ target_link_libraries(signal-estimator
   asound
   m
   ${CMAKE_THREAD_LIBS_INIT}
-  )
-
-install(
-  TARGETS signal-estimator
-  RUNTIME DESTINATION bin
-  )
```

Comment:
Why is this removed?

Reply:
weird, I don't remember removing that. All I did was change it to work with the reorganized src directory.

Reply:
I don't know why it was removed. I only edited the add_executable section to fit the src directory layout. I just added it to the cmakelist.